newsroom/auth_server/oauth2.py

```
@@ -32,7 +33,19 @@
 
 @blueprint.route(TOKEN_ENDPOINT, methods=["POST"])
 def issue_token():
-    return authorization.create_token_response()
+    current_time = utcnow()
+    try:
+        token_response = authorization.create_token_response()
+    except Exception:
+        pass
```

not sure why ignoring the exception?